masterlist.yaml

```diff
@@ -1314,35 +1581,58 @@ plugins:
       - crc: 0x32F4BBE5
         util: SSEEdit v3.2.1
   - name: 'SexLab-AmorousAdventures.esp'
+    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/7305/']
     tag:
       - C.Climate
       - C.Water
   - name: 'ULRiverwoodForestJKPatch.esp'
+  # Unique Locations - Riverwood Forest
+    url: 
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/9258/'
+        name: 'Unique Locations - Riverwood Forest on Nexus Mods'
     tag:
       - C.Water
   - name: 'LADX_SSE.esp'
+  # Legendary Armors - DeserterX Collection
```

Comment:
No need for the comment since you've added a named link.

Reply:
Removed.